apps/stable_diffusion/web/ui/img2img_ui.py

```diff
@@ -77,6 +77,13 @@
 
                 init_image = gr.Image(label="Input Image", type="pil")
 
+                with gr.Accordion(label="Stencil Options", open=False):
+                    with gr.Row():
+                        use_stencil = gr.Dropdown(
+                            label="Stencil model",
+                            value="",
+                            choices=["canny"],
```

Comment:
the default value should be one from the choices. so make this something like (choices = ["None", "canny"] , value="None").